README.rst

```diff
@@ -5,3 +5,12 @@ Provides basic automatic locking support for Plone. Locks are stealable by
 default, meaning that a user with edit privileges will be able to steal
 another user's lock, but will be warned that someone else may be editing
 the same object. Used by Plone, Archetypes and plone.app.iterate
+
+Compatibility
+-------------
+
+The versions 2.1.x (built from the master-branch) are used in Plone 5 and are not compatible with earlier Plone versions.
+
+Versions 2.0.x are used by Plone 4.3.x.
+
+Versions 1.x are ....
```

Comment:
I think it just jumped because of the license change / clarification. 

Plone 336 uses 1.2.1 though